llama-index-core/llama_index/core/evaluation/batch_runner.py

```diff
@@ -378,3 +379,17 @@ def evaluate_queries(
                 **eval_kwargs_lists,
             )
         )
+
+    def upload(
+        self, project_id: str, app_name: str, results: Dict[str, List[EvaluationResult]]
+    ) -> None:
+        """
+        Upload the evaluation results to LlamaCloud
+        """
+        client = get_client()
```

Comment:
tbh i wonder if we can expose get_client directly for users to use (then they can just call `create_local_eval_set_for_project`), but having this method is fine

Reply:
We could do that. Thoughts @logan-markewich?

Reply:
The client UX is very not good. We should optimize the client UX before giving it to users

Reply:
Is there any reason to have this method on the batch eval runner class? Why not have it be a standalone method? Similar to the function that uploads eval datasets to platform

Reply:
We should probably return or print something to indicate success? will add something

Reply:
i agree that a standalone function is probably better

Reply:
why not both :) should be good to go now